### ISSUE TITLE
Feature Request - Hide Archived/Purged Bookmarks

### ISSUE THREAD

**Issue by Efruit** [K1rakishou/Kuroba-Experimental#726]
The bookmarked threads view should expose a "Hide" menu with the following options:

- Hide > Purged: hides, but does not delete, bookmarks of threads that have been purged/deleted

- Hide > Archived: same as the above, but includes bookmarked threads that have fallen in to the on-site archive for 4chan(nel).

- Hide > None: the current behavior, showing all threads regardless of status

**Comment by K1rakishou** [comment]
You can manually move bookmarks into bookmark groups and then just close the groups which will "hide" them.

**Comment by Efruit** [comment]
> You can manually move bookmarks into bookmark groups and then just close the groups which will "hide" them.

Yes you could. You could also manually bookmark threads instead of having a bookmark watcher. The point is that sometimes threads get bookmarked automatically, and I'd like to read them despite them being inactive. Right now there is no reliable method to find threads that may still be available despite being "inactive".

**Comment by Efruit** [comment]
>having to bump github issues

well this is going to be tedious
